src/rp2_common/pico_stdio_usb/stdio_usb.c

```diff
@@ -66,7 +66,7 @@ static void low_priority_worker_irq(void) {
 #endif
         mutex_exit(&stdio_usb_mutex);
 #if PICO_STDIO_USB_SUPPORT_CHARS_AVAILABLE_CALLBACK
-        if (chars_avail && chars_available_callback) chars_available_callback(chars_available_param);
+        if (chars_avail && chars_available_callback) stdio_usb_run_chars_available_callback();
```

Comment:
I think i'd revert this line... no point in calling another method here (which also checks `chars_avail_callback` - id just call it directly to save introduction of the new function (which likely isn't otherwise called when this code is included